static/docs/command-reference/update.md

```diff
@@ -32,9 +32,11 @@ Another detail to note is that when the `--rev` (revision) option of
 `dvc import` has been used to create an import stage, DVC is not aware of what
 kind of
 [Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References) this
-is, for example a branch or a tag. For static refs such as tags (unless manually
-updated), or for SHA commits, `dvc update` will not have any effect on the
-import.
+is, for example a branch or a tag. For static references such as tags (unless
+manually updated), or for SHA commits, `dvc update` will not have any effect on
+the import. Refer to the
+[re-importing example](/doc/command-reference/import#example-fixed-revisions-re-importing)
+to learn how to "update" fixed-revision imports.
```

Comment:
I think we can assume that people understand what tags and branches are and they should get how updating and switching differ. So this paragraph might be shortened.

Or am I wrong? Do people understand switching is a kind of update? If they don't then us mixing these two might confuse instead of helping.

Reply:
I don't understand your comment. No switching mentioned in this paragraph (lines 31-39).

Reply:
Perhaps the text was a little bit confusing? I re-phrased this, please review again.